satellite/marketing/marketingweb/server.go

```diff
@@ -128,20 +163,74 @@ func (s *Server) parseTemplates() (err error) {
 	return nil
 }
 
+// Time converter is used by decoder in formToStruct to format expiration date field from form input.
```

Comment:
Same as above

Reply:
Pushed. Pull down latest to see all updated comment formatting.